app/views/thredded/messageboards/_messageboard.html.erb

```diff
@@ -19,6 +19,8 @@
                 user: messageboard.last_user.thredded_display_name %>
         </p>
       <% end %>
+      <br>
+      <%= inline_svg('thredded/lock.svg', class: 'threadded--messageboard--icon') if messageboard.locked %>
```

Comment:
This tag should have a `title` attribute for accessibility. How about:

> This messageboard is locked. Only moderators can create new topics here.

Reply:
Good point, and that message seems suitable. I didn't change this yet since we'd need a translation node name for it.